app/components/card/CardWithArticle.tsx

```diff
@@ -0,0 +1,37 @@
+import AnimatedButton from "../buttons/AnimatedButton";
+import Card from "./Card";
+
+interface CardWithArticleProps {
+  image: string | null;
+  title: string;
+  linkToArticle: string;
+  createdAt: string;
+}
+
+const CardWithArticle = (props: CardWithArticleProps) => {
+  const { image, title, linkToArticle, createdAt } = props;
+
+  return (
+    <Card>
+      <>
+        <img
```

Comment:
kanskje legge på en altImg prop også? for skjermlesere?

Reply:
Hva tenker du om at denne er det samme som tittelen til artikkelen? Eller er ikke det den riktige måten å gjøre det på?

Reply:
Usikker på om folk bruker å legge til altText på bildene på bloggpostene sine? Men hvis ikke, så kan det være en grei midlertidig løsning, ja :) 

Lurer litt på hvordan vi skal gjøre det med bloggposter da. Det er kanskje litt for tungvint å manuelt legge til lenker til bloggposter i databasen. Kanskje det går an å oppfordre folk til å legge på tags som f.eks. `læreglede` hvis de tenker at det kan havne i denne huben? også kan vi subscribe på den tagen ellernoe? Har du tenkt noe rundt dette?